tf/s3_events.tf

```diff
@@ -0,0 +1,58 @@
+# AWS_DEFAULT_REGION=eu-central-1 terraform plan -var bucket=treeverse-ariels-test  
+
+data "aws_s3_bucket" "bucket" {
+  bucket = var.bucket
+}
+
+# This should properly be a FIFO queue, to avoid mixing up Create and Delete
+# operations.  However AWS says:
+#
+# error putting S3 Bucket Notification Configuration: InvalidArgument: FIFO
+#     SQS queues are not supported.
+resource "aws_sqs_queue" "s3_events_queue" {
+  name = "terminus-s3-events-queue"
```

Comment:
Can terminus be used for more than one bucket? If so, the queue should have a unique name for every bucket

Reply:
Yes it can!  Modified to support multiple buckets.